msfdb

```diff
@@ -204,28 +187,30 @@ end
 
 def stop_db
   update_db_port
-
-  if run_cmd("pg_ctl -o \"-p #{@options[:db_port]}\" -D #{@db} status") == 0
-    puts "Stopping database at #{@db}"
-    run_cmd("pg_ctl -o \"-p #{@options[:db_port]}\" -D #{@db} stop")
-  else
-    puts "Database is no longer running at #{@db}"
-  end
+  @db_driver.stop_db
 end
 
 def restart_db
-  stop_db
-  start_db
+  @db_driver.restart_db
 end
 
 def create_db
-  puts "Creating database at #{@db}"
-  Dir.mkdir(@db)
-  run_cmd("initdb --auth-host=trust --auth-local=trust -E UTF8 #{@db}")
+  @db_driver.init_db
+end
 
-  File.open("#{@db}/postgresql.conf", 'a') do |f|
-    f.puts "port = #{@options[:db_port]}"
-  end
+def create_db_users
+  puts 'Creating database users'
+  run_psql("create user #{@options[:msf_db_user]} with password '#{@msf_pass}'")
+  run_psql("create user #{@options[:msftest_db_user]} with password '#{@msftest_pass}'")
+  run_psql("alter role #{@options[:msf_db_user]} createdb")
+  run_psql("alter role #{@options[:msftest_db_user]} createdb")
+  run_psql("alter role #{@options[:msf_db_user]} with password '#{@msf_pass}'")
+  run_psql("alter role #{@options[:msftest_db_user]} with password '#{@msftest_pass}'")
+
+  conn = PG.connect(host: '127.0.0.1', dbname: 'postgres', port: @options[:db_port], user: @options[:msf_db_user], password: @msf_pass)
+  conn.exec("CREATE DATABASE #{@options[:msf_db_name]}")
+  conn.exec("CREATE DATABASE #{@options[:msftest_db_name]}")
+  conn.finish
```

Comment:
Any context on why these changes were needed in comparison to what was here before?

Reply:
They aren't needed, worked fine the old way, but we have the `pg` gem anyway and I thought this was cleaner, I don't have overly strong feelings one way or the other though so happy to change it back if someone does